protocol.md

```diff
@@ -160,6 +166,7 @@ NoteCutObject = {
 		Number, // Z value
 	],
 	"cutDistanceToCenter": Number, // Distance from the center of the note to the cut plane
+	"cutDistanceInverted": Boolean, // Whether the cut plane distance needs to be mirrored when being represented graphically
```

Comment:
If this property is to exist, I'd rather have the description correctly describe what the flag represents. For example, and this might be incorrect as I'm not sure what things are at the moment, "If true, the cut point is on the left side of the cut normal.".